Exercises/exercise_3_join_validator.md

```diff
@@ -89,13 +106,26 @@ As an Axelar Network validator, your Axelar node will vote on the status of Bitc
     -b block
   ```
 
-  Here `amount` refers to the amount of coins to stake, with a minimum of 5,000,000 axltest. You can change the amount, but leave some coins on the account to fund commands.
-  `moniker` refers to the nickname of your validator. You can give it any nickname you like.
+  To check how many coins your validator has currently staked.
+  ```
+  axelard q staking validator "$(axelard keys show validator --bech val -a)" | grep tokens
+  ```
+
+  If you wish to stake more coins after the initial validator creation.
+  ```
+  axelard tx staking delegate {axelarvaloper address} {amount} --from validator -y
```

Comment:
how does one find axelarvaloper address?

Reply:
The example below automatically gets the address